### ISSUE TITLE
Update Django documentation

### ISSUE THREAD

**Issue by Frzk** [Scalingo/documentation#2279]
It seems we still advise to use `dj-static` in the "Serving Static Files" section (https://doc.scalingo.com/languages/python/django/start#static-file-serving)
`dj-static` is deprecated, so we should update this section and remove it.

**Comment by ksol** [comment]
Indeed. A customer just told me that "whitenoise" is the preferred option, and it works with wsgi and asgi apps

**Comment by Frzk** [comment]
We also had customer that made a PR with `dj-static`, which has been merged.
Maybe it's still popular in the ecosystem?

Consequently, I suggest to:
- Switch to `whitenoise`, since it seems to be the state-of-the-art nowadays.
- Also keep `dj-static`, with a warning (and maybe explaining how  to switch from `dj-static` to `whitenoise`).
- Add a section about `Static`: https://pypi.org/project/static/

**Comment by ksol** [comment]
I'm not familiar at all with the state of the python/django ecosystem, but dj-static has been archived since 2021, so personally I would have removed anything related to it except a disclaimer/warning specifying that whitenoise is the de-facto option and dj-static should not be considered.

Then again, that's not a really informed stance
